src/s1reader/s1_annotation.py

```diff
@@ -0,0 +1,343 @@
+'''
+A module to load annotation files for Sentinel-1 IW SLC SAFE data
+'''
+
+import datetime
+import xml.etree.ElementTree as ET
+import os
+import zipfile
+import glob
+import numpy as np
+
+
+
+class Annotation:
+    '''A parent class for other kinds of .xml files. Contains core attributes and common methods.
+       Accepted format: SAFE .zip file, SAFE directory, or annotation .xml file
+    '''
+    # A parent class for annotation reader for Calibrarion, Noise, and poteltially L1
+    def __init__(self,path_annotation:str,polarization='VH',subswath=1):
+        #Attributes
+        self._ipf_version=None
+        self.path_annotation=path_annotation 
+        self.polarization=polarization
+        self.subswath=subswath #Subswath of the annotation file to load
```

Comment:
`ipf_version` is more appropriate as an attribute of `Sentinel1BurstSlc`. `polarization` and `subswath` are already members of `Sentinel1BurstSlc`. If these and file finding functionality are removed, then there's no need for class inheritance here?

Reply:
again, the attribute `ipf_version` is added into `Sentinel1BurstSLC`. The annotation class will not extract the subswath and polarization as it will rely on the arguments provided when calling `s1_reader.load_bursts()`